docs/notebooks/model-levels.ipynb

```diff
@@ -0,0 +1,2052 @@
+      ],
+      "text/plain": [
+       "MetDataset with data:\n",
+       "\n",
+       "<xarray.Dataset>\n",
+       "Dimensions:                              (time: 18, level: 23, latitude: 181,\n",
+       "                                          longitude: 360)\n",
+       "Coordinates:\n",
+       "  * time                                 (time) datetime64[ns] 2024-01-15 ......\n",
+       "    step                                 timedelta64[ns] 00:00:00\n",
+       "  * level                                (level) float64 170.0 180.0 ... 390.0\n",
+       "  * latitude                             (latitude) float64 -90.0 -89.0 ... 90.0\n",
+       "    valid_time                           (time) datetime64[ns] 2024-01-15 ......\n",
+       "  * longitude                            (longitude) float64 -180.0 ... 179.0\n",
+       "    air_pressure                         (level) float32 1.7e+04 ... 3.9e+04\n",
+       "    altitude                             (level) float32 1.281e+04 ... 7.364e+03\n",
+       "Data variables:\n",
+       "    air_temperature                      (longitude, latitude, level, time) float32 ...\n",
+       "    specific_humidity                    (longitude, latitude, level, time) float32 ...\n",
+       "    eastward_wind                        (longitude, latitude, level, time) float32 ...\n",
+       "    northward_wind                       (longitude, latitude, level, time) float32 ...\n",
+       "    lagrangian_tendency_of_air_pressure  (longitude, latitude, level, time) float32 ...\n",
+       "    specific_cloud_ice_water_content     (longitude, latitude, level, time) float32 ...\n",
+       "Attributes:\n",
+       "    provider:  ECMWF\n",
+       "    dataset:   ERA5\n",
+       "    product:   reanalysis"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Ready the Dataset for pycontrails\n",
+    "met = MetDataset(\n",
+    "    ds.rename(isobaricInhPa=\"level\"),\n",
+    "    provider=\"ECMWF\",\n",
+    "    dataset=\"ERA5\",\n",
+    "    product=\"reanalysis\",\n",
+    ")\n",
+    "met.standardize_variables(PRESSURE_LEVEL_VARIABLES)\n",
+    "met"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Missing values\n",
+    "\n",
+    "Converting from model levels to pressure levels will result in missing values. Specifically, points at which the surface pressure is low result in missing values at the lowest target pressure levels. Additional higher model levels can be downloaded to decrease the number of missing values."
```

Comment:
"Higher model levels" is a bit confusing: does it mean higher above the surface or a level with higher pressure? Maybe worth clarifying that you mean the latter.

Reply:
I tweaked this a little: 8fc22b921c9a9090ee39bf92675c355c9704305c